src/Header/Account/Account.jsx

```diff
@@ -148,16 +175,18 @@ export default class Account extends Component {
 			</div>
 		) : (
 			<a
-				href={signinUrl}
+				href={signInLink.value}
 				className={styles.button}
 				onClick={this.handleMenuItemClick}
 			>
-				Sign in
+				{signInLink.key}
```

Comment:
On reflection after reading this, I think it probably makes more sense for the key property to be called text and the value one to be called url or href. The fact they're used as key/value pairs in the nice accounts object is coincidental.